components/formats-gpl/utils/ReadWriteInMemory.java

```diff
@@ -35,6 +35,8 @@
 
 /**
  * Tests the Bio-Formats I/O logic to and from byte arrays in memory.
+ * @deprecated Class has been moved to documentation examples and is available for download 
+ * from http://www.openmicroscopy.org/site/support/bio-formats/developers/in-memory.html
  */
 public class ReadWriteInMemory {
```

Comment:
Should the class also be annotated with `@Deprecated` in addition to the Javadoc?